src/Features/LanguageServer/Protocol/Handler/InlineCompletions/InlineCompletionsHandler.cs

```diff
@@ -22,6 +20,7 @@
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 using Roslyn.Utilities;
+using static Microsoft.CodeAnalysis.LanguageServer.Handler.InlineCompletions.XmlSnippetParser;
```

Comment:
Seems unnecessary

Reply:
a couple of types from the parser are used here